.github/workflows/verdi.sh

```diff
@@ -37,3 +37,18 @@ done
 
 $VERDI devel check-load-time
 $VERDI devel check-undesired-imports
+
+
+# Test that we can also run the CLI via `python -m aiida`,
+# that it returns a 0 exit code, and contains the expected stdout.
+OUTPUT=$(python -m aiida 2>&1)
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    echo "'python -m aiida' exitted with code $retVal"
```

Comment:
Why not just `exit 2` here as well? Is there a reason for forwarding the exit code from the test script? Seems slightly unexpected to me.

Reply:
Well my thinking was that the 2nd check, of the stdout, should also run, even if the exit code is not 0

Reply:
Why? But even so, I would still `exit 2` on any failure. It just seems unintuitive to me that the test script would forward the exit code unless explicitly requested.